socorro/external/postgresql/connection_context.py

```diff
@@ -49,6 +51,13 @@ class ConnectionContext(RequiredConfig):
         reference_value_from='secrets.postgresql',
         secret=True,
     )
+    required_config.add_option(
+        name='database_url',
+        default=None,
```

Comment:
rather than set the default to none, this is where you should fetch the value from the environment. 
```python
    import os
    required_config.add_option(
        name='database_url',
        default=os.environ('database_url', None),
        # ...
```